ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/codes/RespConstantsV7.java

```diff
@@ -20,6 +20,21 @@
 
 public final class RespConstantsV7 {
 
+    public static final String IDENTITET_CATEGORY_ID = "identitet";
+    public static final String IDENTITET_CATEGORY_TEXT_ID = "KAT_100.RBK";
+
+    public static final String IDENTITET_STYRKT_GENOM_SVAR_ID_2 = "2";
```

Comment:
I dont really mind, but should we remove the number followed by ID to better match the pattern of our constants?